WORKSPACE

```diff
@@ -28,6 +28,7 @@ yarn_install(
     name = "npm",
     package_json = "//:package.json",
     yarn_lock = "//:yarn.lock",
+    # frozen_lockfile = False,
```

Comment:
Needed to update the yarn lock, leaving it commented because it took me a while to figure out how to get rid of --frozen_lockfile